### PR TITLE
Suggested fix for #254, bad patient_ids

### LINKED ISSUE #254: Eclipse recognizes every DICOM set generated by pytrip as the same patient
matchingname
Here Eclipse thinks that the pytrip-generated DICOM set of Alinas patient from #245  is the same as the test files sent by Leszek and Niels  in regard to #193 (which is clearly not the case).  
The patient ID1 is always 123456 for DICOM sets generated with pytrip. Therefore Eclipse thinks it is the same patient. This seems to happen no matter what the other UIDs say because clearly they are listed with different names ("tst002" vs "cube"), being the original names of the respective .hed file.

This issue makes it a bit difficult to joggle with more than one pytrip-generated DICOM set at a time, because you have to delete the existing patient first in order to import the new one.

### PR DESCRIPTION
**Pull request by nbassler**
Closes #254

### CODE REVIEW

File: .gitattributes [hunk 1]
```diff
@@ -0,0 +1 @@
+pytrip/_version.py export-subst
```
Comment (nbassler) [inline .gitattributes]: @grzanka something not ok in ./versioneer_install

Comment (grzanka) [inline .gitattributes]: I'll fix it.

**Comment by grzanka** [comment]
Patient ID is used both in Cube and in VDX, see: https://github.com/pytrip/pytrip/blob/master/pytrip/vdx.py#L272

**Comment by grzanka** [comment]
@nbassler I did a "push force" on this branch to rebase it and resolve conficts.
Please remove local branch and get fresh remote if you want to add sth.

**Pull request merged**